src/App.jsx

```diff
@@ -1,14 +1,18 @@
-import React, { useState, useEffect } from 'react';
```

Comment:
from react .... not sure 16? on... we don't need to import explicitely `React` itself and we can just import the functions we need.